vac/raw/eth-secpm.md

```diff
@@ -824,6 +824,33 @@ These credentials MUST use the digital signature key pair associated to the Ethe
 - A Delivery Service storage system MUST verify KeyPackages before storing them.
 - Interaction diagrams involving the DS do not change.
 
+## Consideration related to the onchain component of the protocol
+- Each group is managed by a separate smart contract.
+
+- Below follows a description of Alice adding Bob as a member of a new group.
+    1. Alice creates the smart contract associated to the group. This smart contract MUST include an ACL.
+    2. Alice adds Bob’s Ethereum address to the ACL. 
+    3. Off-chain: Alice and Bob set a secure communication channel. This MUST be done following section *Private 1-to-1 communications protocol*
+    4. Off-chain: Bob sends a request to Alice so he can be included in the group. This request is a digitally signed message. **The signature MUST be generated using Bob’s secret key, associated to his Ethereum address.
```

Comment:
What data we wanna include into request? generally speaking, at this point we should already be sending a key package. 

In term of openmls the key package will already include a signature and a public key for verification and in effect this eliminates the use of a smart contract. 
That is, at this stage of implementation we will have two signatures - one inside the key package and one on top using the eth wallet key